src/main/java/net/mcreator/ui/blockly/BlocklyPanel.java

```diff
@@ -103,9 +103,8 @@ public BlocklyPanel(MCreator mcreator) {
 						css += FileIO.readResourceToString("/blockly/css/mcreator_blockly_unixfix.css");
 					}
 
-					if (PluginLoader.INSTANCE
-							.getResourceAsStream("/themes/" + ThemeLoader.CURRENT_THEME.getID() + "/styles/blockly.css")
-							!= null) {
+					if (FileIO.readResourceToString(PluginLoader.INSTANCE,

```

Comment:
readResourceToString will never return null, though, so this is not ok either. Previous code should work as if resource is not present, it will return null

Reply:
Previous code returned me null even if the file was present.

Reply:
Hmm I will take a look

Reply:
I have properly fixed this in master. Please revert this part of PR